src/DataProtection/samples/EntityFrameworkCoreSample/Migrations/20181219214311_AddDataProtectionKeys.Designer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <auto-generated />
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using EntityFrameworkCoreSample;
+
+namespace EntityFrameworkCoreSample.Migrations
+{
+    [DbContext(typeof(DataProtectionKeyContext))]
+    [Migration("20181219214311_AddDataProtectionKeys")]
+    partial class AddDataProtectionKeys
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "2.2.0-rtm-35687")
```

Comment:
This seems odd. Samples in our master branch should be using EFCore 3.0. Which version of the dotnet CLI did you use to generate this migration?

Reply:
2.2.100 ... but I had the preview on here earlier.

Reply:
Ok. I'm fine as is. This is "just a sample" after all, and as long as EFCore doesn't break the migrations API in 3.0, there's no real concern here.